frontend/src/scenes/insights/PersonModal.js

```diff
@@ -24,9 +27,21 @@ export function PersonModal({ visible, view }) {
             width={800}
         >
             {people ? (
-                <p>
+                <div
+                    style={{ marginBottom: 16, display: 'flex', alignItems: 'center', justifyContent: 'space-between' }}
+                >
                     Found {people.count === 99 ? '99+' : people.count} {people.count === 1 ? 'user' : 'users'}
-                </p>
+                    {user?.is_multi_tenancy ? (
+                        <div>
+                            <LinkButton to={peopleModalURL.recordings} type="primary" target="_blank">
```

Comment:
We def need to add instrumentation here, e.g. `data-attr="persons-modal-recordings`